src/cljs/villagebookUI/components/new_org.cljs

```diff
@@ -0,0 +1,34 @@
+(ns villagebookUI.components.new-org
+  (:require [reagent.core :as r]
+            [villagebookUI.helpers :refer [random-color handle-esc]]
+            [villagebookUI.components.utils :as utils]
+            [villagebookUI.api.organisation :as org]))
+
+(defn create-org [form success-handler error-handler]
+  (when (:name form)
+    (org/create-org form
+                    success-handler
+                    error-handler)))
+
+(defn new-org [on-close-handler]
```

Comment:
Would `org-creation-form` be a better name?

Reply:
Yes, makes more sense than `new-org`. Something like `new-org-form` should work too.